modules_core/org.openbravo.advpaymentmngt/src/org/openbravo/advpaymentmngt/actionHandler/AddPaymentActionHandler.java

```diff
@@ -82,166 +83,223 @@
 import org.openbravo.service.db.DbUtility;
 import org.openbravo.service.json.JsonUtils;
 
-public class AddPaymentActionHandler extends BaseProcessActionHandler {
+import com.smf.jobs.Action;
+import com.smf.jobs.ActionResult;
+
+public class AddPaymentActionHandler extends Action {
   private static final Logger log = LogManager.getLogger();
 
+
+
+
+
+//  @Override
```

Comment:
Should these commented lines of code be deleted or are they useful? If so, then uncomment them